### PR TITLE
Add example to nss-mymachines(8)

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/nss-mymachines.xml [hunk 1]
```diff
@@ -35,12 +35,20 @@
     <para><command>nss-mymachines</command> is a plug-in module for the GNU Name Service Switch (NSS) functionality of
```
Comment (poettering) [inline man/nss-mymachines.xml:35]: the commit msg is a bit misleading. it's nss-mymachines, not nss-machines...

File: man/nss-mymachines.xml [hunk 2]
```diff
@@ -35,12 +35,20 @@
...
-    container name, and back. This functionality only applies to containers using user namespacing.</para>
+    functionality only applies to containers using network namespacing (see the description of
+    <option>--private-network</option> in
+    <citerefentry><refentrytitle>systemd-nspawn</refentrytitle><manvolnum>1</manvolnum></citerefentry>).
+    Note that the name that is resolved is the one registered with <command>systemd-machined</command>, which
+    may be different than the hostname configured inside of the container.</para>
+
+    <para>The module also provides name resolution for user and group identifiers mapped to containers. All names from
+    the range allocated to a given container <replaceable>container</replaceable> are exposed on the host with the
+    <literal>vu-<replaceable>container</replaceable></literal> and
```
Comment (poettering) [inline man/nss-mymachines.xml]: should maybe say `<literal>vu-<replaceable>container</replaceable>-<replaceable>uid</replaceable></literal>`?

**Comment by poettering** [comment]
(there's als #10110)

**Comment by keszybz** [comment]
Updated with the requested changes.

**Pull request merged**